.github/workflows/release_artifacts.yaml

```diff
@@ -64,7 +64,7 @@ jobs:
         runs-on: ubuntu-latest
 
         container:
-            image: ghcr.io/project-chip/chip-build-efr32:77
```

Comment:
will this be required or nice to have ? Asking because cloud-build also builds efr32 so it may need updating.

Reply:
Specifically I think you should be updating all the versions of the vscode image in https://github.com/project-chip/connectedhomeip/blob/master/integrations/cloudbuild/smoke-test.yaml

Reply:
updated!